### PR TITLE
Tutorial on Scheme graphics

### PR DESCRIPTION
**Pull request by pireddag**
Gradirei commenti, un paio di parti vanno messe a posto.

### CODE REVIEW

File: src/contribution-guide.tm [hunk 1]
```diff
@@ -166,6 +166,7 @@
   <\collection>
     <associate|auto-1|<tuple|?|?>>
     <associate|auto-2|<tuple|?|?>>
+    <associate|auto-3|<tuple|?|?>>
```
Comment (pireddag) [inline src/contribution-guide.tm:169]: Here mergetool forced me

**Comment by pireddag** [comment]
Overall it needs a good introduction. The level and the sequence of steps seem to me ok.

**Review by mgubi — CHANGES_REQUESTED** [review]
I added the header and a comment. Seems good to me. Maybe resolve what you want to say about the part marked in red.

**Comment by pireddag** [comment]
I have added an introduction that looks ok to me
I would need help with these two points, on which in the note I make statements which may or may not be correct:

1. I write "point() is a TeXmacs graphics primitive that represents a point and expects two strings": which types does TeXmacs expect? Maybe the conversion to string in Scheme is necessary only because of the subsequent stree->tree?
2. The Scheme expression that starts with (graphics : this is a Scheme tree, but I do not have clear what is the status in Scheme of the symbol graphics: is it a function, or only a symbol? I would think only a symbol, but I would like confirmation.

**Comment by mgubi** [comment]
> I have added an introduction that looks ok to me
> I would need help with these two points, on which in the note I make statements which may or may not be correct:
> 
> 1. I write "point() is a TeXmacs graphics primitive that represents a point and expects two strings": which types does TeXmacs expect? Maybe the conversion to string in Scheme is necessary only because of the subsequent stree->tree?
> 2. The Scheme expression that starts with (graphics : this is a Scheme tree, but I do not have clear what is the status in Scheme of the symbol graphics: is it a function, or only a symbol? I would think only a symbol, but I would like confirmation.

I've added a remark. As I understand "point" is a primitive in the markup language so there you are forming an stree which will be then converted to a texmacs tree, therefore numbers should be entered as string. (like other content in strees). "graphics" is also markup. That is with you have to put the quote before the full expression. All is a list which is passed to `stree->tree`to form a internal tree and be typesetted.

**Comment by pireddag** [comment]
Please take a look now, I may have overwritten your remark with mergetool, but it should be closer to acceptable now.
There is an additional question on my side: how do I get the expression (graphics,) to appear in blue (as inactive TeXmacs macros do)? If I type \graphics and press enter I get immediately the canvas.
If this is not possible, I will display it as \graphics (maybe say that it is not possible to show it in inactive form)

**Comment by pireddag** [comment]
From my point of view it is ok now, save for the issue of getting \graphics to appear as an inactive macro

**Review by mgubi — APPROVED** [review]
For me now is ok. I will merge this. Please look at how I use markup, you have \scm for scheme code and \markup for TeXmacs markup.

**Comment by mgubi** [comment]
Also file names would be better to have only in lower-case (MacOS is case-insensitive anyway)

**Comment by pireddag** [comment]
Thanks for the info on \scm and \markup (inactive macros were not good). I will read it again tomorrow for possible refinements.

**Pull request merged**